core/src/main/java/com/sequenceiq/cloudbreak/conf/CachingConfig.java

```diff
@@ -17,7 +21,9 @@
 @EnableAutoConfiguration
 public class CachingConfig implements CachingConfigurer {
 
-    public static final int MAX_ENTRIES = 1000;
+    public static final String TEMPORARY_AWS_CREDENTIAL_CACHE = "temporary_aws_credential";
```

Comment:
Why the cache is defined in the core? Shall be just aws related, but as I wrote a generic credential caching service wouldbe better.